src/app/trainee/trainee-details/trainee-details.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="trainee-details" [ngClass]="{'large-display': (isHandset$ | async) === false}"
+    *ngIf="revalidationHistory$ | async as revalidationHistory">
+    <mat-list>
+        <mat-list-item *ngIf="revalidationHistory.traineeName">
+            <div class="mat-caption" mat-line>Name:</div>
+            <div mat-line>{{revalidationHistory.traineeName}}</div>
+        </mat-list-item>
+        <mat-list-item *ngIf="revalidationHistory.gmcId">
+            <div class="mat-caption" mat-line>GMC No:</div>
+            <div mat-line>{{revalidationHistory.gmcId}}</div>
+        </mat-list-item>
+        <mat-list-item *ngIf="revalidationHistory.programmeMembershipType">
+            <div class="mat-caption" mat-line>Programme membership type:</div>
+            <div mat-line>{{revalidationHistory.programmeMembershipType}}</div>
+        </mat-list-item>
+        <mat-list-item *ngIf="revalidationHistory.currentGrade">
+            <div class="mat-caption" mat-line>Current grade:</div>
+            <div mat-line>{{revalidationHistory.currentGrade}}</div>
+        </mat-list-item>
+        <mat-list-item *ngIf="revalidationHistory.cctDate">
+            <div class="mat-caption" mat-line>CCT date:</div>
+            <div mat-line>{{revalidationHistory.cctDate | date: 'dd/MM/yyyy'}}</div>
```

Comment:
would be good if the date string format could be in a global constant somewhere

Reply:
could do, just copied and pasted from angular documentation. any idea if we should add this constant to shared or env variable?

Reply:
After looking at the localization docs, I think this would be a good candidate for i18n en-GB date format. One for future thoughts. I'll stick in environments for now

Reply:
yea localization would defo be a good idea but that feels like its own ticket. i think for now a variable of some sort would be great. i did create `src/app/core/trainee/constants.ts` but this variable feels like it should be somewhere generic/global